perceval/__init__.py

```diff
@@ -66,3 +67,16 @@ def register_plugin(name, silent=False):
     except Exception as e:
         raise RuntimeError("cannot import %s: %s" % (name, str(e)))
     return True
+
+
+def set_logger(use_python_logger: bool = False):
```

Comment:
This function shouldn't be here but in the logging package.

And the naming is not readable. Do you expect people to type `pcvl.set_logger(True)` and understand what this means?

I suggest the following naming: pcvl.utils.logging.use_python_logger() and pcvl.utils.logging.use_perceval_logger()